board_client/animations/lightning.py

```
@@ -3,88 +3,99 @@
-            self.end_y = end_y
-            self.angle = math.atan2(end_y - start_y, end_x - start_x)
+            self.hue_shift = random.uniform(-0.1, 0.1)  # More purple/blue shift
+
+            self.current_x = float(start_x)
+            self.current_y = float(start_y)
             self.path_points = self._plan_path()
 
         def _plan_path(self):
-            # plan lightning path
             points = []
             x, y = self.current_x, self.current_y
```

Ruff (0.11.9)

Local variable `x` is assigned to but never used

Remove assignment to unused variable `x`

(F841)